extensions/wikia/Wall/WallThread.class.php

```diff
@@ -109,13 +113,25 @@ private function loadReplyIdsFromDB($master = false) {
 			$list[] = $row->comment_id;
 		}
 
-		$this->setReplies( $list );
+		$lastId = end( $list );
+
+		return empty( $lastId ) ? $list :
+			array_merge( $list, $this->getReplyIdsFromDB( $dbr, $lastId ) );
```

Comment:
With a limit of 500 rows, this will very rarely ever enter recursion, and when it does, it will be less than 10 recursive calls. I don't think this particular case is anything to worry about.